src/main/java/com/khoros/twitterapp/services/TwitterService.java

```diff
@@ -35,39 +35,27 @@ public TwitterService(Twitter twitterFactory) {
 
         logger.info("Attempting to update status through Twitter API.");
 
-        if (StringUtils.isEmpty(statusText)) {
-
-            logger.info("Twitter status update unsuccessful.");
+        verifyTextLength(statusText);
 
-            throw new TwitterServiceException(TwitterService.NO_TWEET_TEXT_MSG);
+        Optional<Status> responseOptional;
```

Comment:
create this when you need it

Reply:
If I move this declaration the code won't work. This is where I need it. There's a try catch below this declaration. responseOptional is set within the try brackets.